shanoir-ng-front/src/app/shared/components/entity/entity-list.component.abstract.ts

```diff
@@ -115,12 +118,22 @@ export abstract class EntityListComponent<T extends Entity> implements OnDestroy
     }
 
     protected openDeleteConfirmDialog = (entity: T) => {
-        this.confirmDialogService
-            .confirm(
-                'Delete ' + this.ROUTING_NAME,
-                'Are you sure you want to delete the ' + this.ROUTING_NAME
-                + (entity['name'] ? ' "' + entity['name'] + '"' : ' with id n° ' + entity.id) + ' ?'
-            ).then(res => {
+        let dialogTitle : string = 'Delete ' + this.ROUTING_NAME;
+        let dialogMsg : string = 'Are you sure you want to delete the ' + this.ROUTING_NAME
```

Comment:
@pierrehenri-dauvergne please change: "Are you sure you want to finally delete the "